### PR TITLE
Objects Automated Test - LPS-156694, LPS-156695, LPS-156699, LPS-156700 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-156694
https://issues.liferay.com/browse/LPS-156695 (test with quarantine label)
https://issues.liferay.com/browse/LPS-156699
https://issues.liferay.com/browse/LPS-156700

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

**Review by timpak — APPROVED** [review]
Really great job with this tests @andreefarias! LGTM!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase [hunk 1]
```diff
@@ -713,6 +713,77 @@ definition {
 		}
 	}
 
+	@description = "LPS-147651 - Verify that the user can define and use a expression with 'Is Empty' to text fields"
+	@priority = "4"
+	test CanDefineExpressionIsEmpty {
+		property portal.upstream = "quarantine";
+
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase:720]: @andreefarias in the future, you should quarantine the test in a separate commit so it's easier to track the history of the test. Also, please create a ticket to remove the test from quarantine and link it to the automation ticket.

**Comment by timpak** [comment]
See https://github.com/timpak/liferay-portal/pull/308

**Pull request closed without merging**